dist/mapd-crossfilter.js

```diff
@@ -19447,11 +19447,15 @@ function replaceRelative(sqlStr) {
         return group;
       }
 
-      function value(ignoreFilters, ignoreChartFilters, callback) {
+      function value(ignoreFilters, ignoreChartFilters, ignoreFilterIndex, callback) {
+        // for backward compatibility
```

Comment:
This might be sliiiiightly safer as a typeof = function check instead, but I'm assuming that all call points are covered by this already anyway.